youtube_dl/YoutubeDL.py

```diff
@@ -1524,7 +1524,7 @@ def dl(name, info):
                         fd.add_progress_hook(ph)
                     if self.params.get('verbose'):
                         self.to_stdout('[debug] Invoking downloader on %r' % info.get('url'))
-                    return fd.download(name, info)
+                    return fd.download(name, info, self.params.get("md5"))
```

Comment:
We know about post process changes. But my teammates who directly use ydl have also harddrive issusa caus use it in 20-30 threads. It is ok if i write pre and postprocess md5 calculation?